tester/runner.go

```diff
@@ -600,7 +600,6 @@ func Load(args []string, filter loader.Filter) (map[string]*ast.Module, storage.
 	err = storage.Txn(ctx, store, storage.WriteParams, func(txn storage.Transaction) error {
 		for _, loadedModule := range loaded.Modules {
 			modules[loadedModule.Name] = loadedModule.Parsed
-
```

Comment:
nit: I think we can add this line back since it's an unrelated change 🙂

Reply:
Sure